examples/bri-3/src/bri/communication/agents/messaging.agent.ts

```diff
@@ -57,11 +65,11 @@ export class MessagingAgent implements OnApplicationBootstrap {
     );
   }
 
-  public validateBpiMessageFormat(
+  public tryDeserializeToBpiMessageCandidate(
     rawMessage: string,
   ): [CreateBpiMessageDto, string[]] {
     const errors: string[] = [];
-    let newBpiMessageCandidate: CreateBpiMessageDto;
+    let newBpiMessageCandidate: CreateBpiMessageDto; // TODO: Switch to BpiMessage instead of DTO
```

Comment:
can we fix this todo before merging this?

Reply:
fixed